RawWaterManagement/SluiceGate/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,142 @@
+  },
+  "id": "urn:ngsi-ld:dataModel:id:EGUP:16321644",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2003-06-19T06:37:36Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2019-02-20T21:54:19Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Else standard campaign never."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Military successful report wear necessary let research. Decade choice cell."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Cost tell site concern cost. West whom hope feel nothing candidate political. Government knowledge dog inside."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Too commercial a step. Bill another choice next suffer. Generation bank spend happen police."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Than produce spend executive talk fall relationship. Place fire require each up yet participant."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:RVGI:73108415",
+      "urn:ngsi-ld:dataModel:items:BKDJ:84525829"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:UREZ:52029940",
+      "urn:ngsi-ld:dataModel:items:ZMTO:27892388"
+    ]
+  },
+  "type": "SluiceGate",
+  "tag": {
+    "type": "Property",
+    "value": "Leg great small every wind the really. Its safe bad condition break nation value participant."
+  },
+  "gateType": {
+    "type": "Property",
+    "value": "Sluice Gate"
+  },
+  "gateWidth": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "gateOpening": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "gateBottomElevation": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "gateDischargeCoefficient": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "orificeDischargeCoefficient": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "waterDischarge": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "headDifference": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "flowType": {
+    "type": "Property",
+    "value": "Submerged Flow"
+  },
+  "upstreamEndControlPoint": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:upstreamEndControlPoint:JXFD:60487647"
+  },
+  "downstreamEndControlPoint": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:downstreamEndControlPoint:CBWI:21948924"
+  },
+  "upstreamControlPoint": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:upstreamControlPoint:MWGU:81565938"
+  },
+  "downstreamControlPoint": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:downstreamControlPoint:GIWE:80160975"
+  },
+  "observedBy": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:observedBy:ZWZM:93328711"
+  },
+  "curveDischargeCoefficient": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}